pillar/certificates.sls

```diff
@@ -14,3 +14,37 @@ certificate_information:
   days_remaining:
     ca_certificate: 90
     certificate: 90
+
+ssl:
```

Comment:
Maybe we could remove this top `ssl` name by something more consistent: `certs_paths`, `certificates_paths`...